vite.config.mts

```diff
@@ -36,8 +37,9 @@ export default defineConfig({
   envPrefix: ['REACT_APP_'],
   define: {
     // https://github.com/bevacqua/dragula/issues/602#issuecomment-1296313369
-    global: 'globalThis',
+    // global: 'globalThis',
```

Comment:
Explanation:
Likely fixes something for a dev-build, breaks building prod.

Reply:
what error are you getting?

Reply:
Well, I can't remember exactly.
Weird thing is that if I now reset back to your last commit, I also am unable to replicate

it was something with a dependency's import where globalThis got applied to the 'from' string breaking the build

Reply:
![image](https://github.com/GoodDollar/GoodDAPP/assets/6606028/66d5401f-231c-4cf1-aa16-efd80bd76a03)

ah okay. apparently I deleted yarn lock the first time. thats when above issue happens